sdk/storage/storage-blob-changefeed/review/storage-blob-changefeed.api.md

```diff
@@ -0,0 +1,85 @@
+## API Report File for "@azure/storage-blob-changefeed"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BlobServiceClient } from '@azure/storage-blob';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+
+// @public (undocumented)
+export class BlobChangeFeedClient {
+    constructor(blobServiceClient: BlobServiceClient);
+    // (undocumented)
+    getChanges(options?: ChangeFeedGetChangesOptions): PagedAsyncIterableIterator<BlobChangeFeedEvent, BlobChangeFeedEventPage>;
+    }
+
+// @public (undocumented)
+export interface BlobChangeFeedEvent {
+    // (undocumented)
+    data: BlobChangeFeedEventData;
+    // (undocumented)
+    dataVersion?: string;
+    // (undocumented)
+    eventTime: Date;
+    // (undocumented)
+    eventType: BlobChangeFeedEventType;
+    // (undocumented)
+    id: string;
+    // (undocumented)
+    metadataVersion: string;
+    // (undocumented)
+    subject: string;
+    // (undocumented)
+    topic: string;
+}
+
+// @public (undocumented)
+export interface BlobChangeFeedEventData {
+    // (undocumented)
+    api: string;
+    // (undocumented)
+    blobType: BlobType;
+    // (undocumented)
+    clientRequestId: string;
+    // (undocumented)
+    contentLength: number;
+    // (undocumented)
+    contentType: string;
+    // (undocumented)
+    etag: string;
+    // (undocumented)
+    requestId: string;
+    // (undocumented)
+    sequencer: string;
+    // (undocumented)
+    url: string;
+}
+
+// @public (undocumented)
+export class BlobChangeFeedEventPage {
+    constructor();
+    // (undocumented)
+    continuationToken: string;
+    // (undocumented)
+    events: BlobChangeFeedEvent[];
+}
+
+// @public (undocumented)
+export type BlobChangeFeedEventType = "UnspecifiedEventType" | "BlobCreated" | "BlobDeleted" | "BlobPropertiesUpdated" | "BlobSnapshotCreated" | "Control" | "BlobTierChanged" | "BlobAsyncOperationInitiated" | "BlobMetadataUpdated";
+
+// @public (undocumented)
+export type BlobType = "BlockBlob" | "AppendBlob" | "PageBlob";
+
+// @public (undocumented)
+export interface ChangeFeedGetChangesOptions {
```

Comment:
BlobChangeFeedGetChangesOptions? Everything else is prefixed with BlobChangeFeed, though I realize this makes a long type even longer.

Reply:
Okay.